source/features/convert-release-to-draft.tsx

```diff
@@ -52,7 +52,7 @@ function attachButton(editButton: HTMLAnchorElement): void {
 	editButton.before(
 		<button
 			type="button"
-			className="btn btn-sm ml-3 mr-1 rgh-convert-draft"
+			className="Button Button--secondary Button--small ml-3 mr-1 rgh-convert-draft"
```

Comment:
```suggestion
			className="Button Button--secondary Button--small ml-2 rgh-convert-draft"
```

| Before | After |
| --- | --- |
| <img width="575" height="123" alt="Image" src="https://github.com/user-attachments/assets/5ec8cf6d-1790-480b-95f4-d460841cd864" /> | <img width="559" height="155" alt="Image" src="https://github.com/user-attachments/assets/5d4f8f17-5b51-450e-b846-18285b361349" /> |